dev/benchmarks/complex_layout/windows/runner/win32_window.cpp

```diff
@@ -4,12 +4,17 @@
 
 #include "win32_window.h"
 
+#include <dwmapi.h>
 #include <flutter_windows.h>
 
 #include "resource.h"
 
 namespace {
 
+#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
+#define DWMWA_USE_IMMERSIVE_DARK_MODE 20
+#endif
```

Comment:
This was added in Windows 11 Build 22000 ([link](https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute)). Developers that haven't updated to the latest Windows SDK may not have these defined in their version of `dwmapi.h`. These manual defines prevent these kinds of issues, recommended by: https://docs.microsoft.com/windows/apps/desktop/modernize/apply-windows-themes#enable-a-dark-mode-title-bar-for-win32-applications

Reply:
Add a TODO to remove this define when we bump our Windows SDK on the bots.

Reply:
Sorry since this is code that lives on the developer's machine, this is going to be dependent on which SDK they have installed. This is probably fine as-is, but add a comment about why we're defining it here.

Reply:
Added a comment!